config/database.yml

```diff
@@ -0,0 +1,38 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: sqlite3
+  # adapter: postgresql
```

Comment:
- [ ] Since it is a requirement for this project, please make sure your `Database adapter is using Postgresql`. See the screenshot of the requirements below for clarifications and kindly fix that.

![postgres](https://user-images.githubusercontent.com/62849872/154430287-f77b8a4c-6b6e-4e82-aa2c-9170f9b81a27.png)